packages/cli/package.json

```diff
@@ -37,6 +37,7 @@
     "fs-extra": "^9.0.1",
     "live-server": "1.2.1",
     "lodash": "^4.17.15",
+    "material-icons": "^1.9.1",
```

Comment:
Can we remove the `material-icons` dependency from here since it does not seem to be used in this package? Same for `core-web` and `vue-components`.